paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/manage/ManageSavedPaymentMethodMutatorFactory.kt

```diff
@@ -39,7 +41,12 @@ internal class ManageSavedPaymentMethodMutatorFactory @Inject constructor(
                 selectionHolder.set(null)
             },
             customerStateHolder = customerStateHolder,
-            prePaymentMethodRemoveActions = {},
+            prePaymentMethodRemoveActions = {
+                if (customerStateHolder.paymentMethods.value.size > 1) {
```

Comment:
Caused by #10107 

Will have integration tests on the activity to validate this doesn't regress again.